hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/hevm/HederaEvmTransactionResult.java

```diff
@@ -282,6 +291,11 @@ public boolean isSuccess() {
         return stateChangesFrom(frame, false);
     }
 
+    private static String errorMessageForRevert(@NonNull final Bytes reason) {
+        requireNonNull(reason);
+        return "0x" + reason.toHex();
```

Comment:
ooookay ... why?  (someone external must need it?)

Reply:
Haha backward compatibility, I have no clear sense why we are externalizing these in hex 😵‍💫